code/game/objects/structures/crates_lockers/closets/gimmick.dm

```diff
@@ -47,6 +47,22 @@
 	new /obj/item/clothing/under/soviet(src)
 	new /obj/item/clothing/under/soviet(src)
 	new /obj/item/clothing/under/soviet(src)
+	new /obj/item/clothing/head/beret (src)
+	new /obj/item/clothing/head/beret (src)
+	new /obj/item/clothing/head/beret (src)
+	new /obj/item/clothing/head/beret (src)
+	new /obj/item/clothing/head/beret (src)
+	new /obj/item/clothing/head/beret (src)
+	new /obj/item/clothing/head/bearpelt (src)
+	new /obj/item/clothing/head/bearpelt (src)
+	new /obj/item/clothing/head/bearpelt (src)
+	new /obj/item/clothing/head/bearpelt (src)
+	new /obj/item/clothing/head/bearpelt (src)
+	new /obj/item/clothing/shoes/steeltoe (src)
+	new /obj/item/clothing/shoes/steeltoe (src)
+	new /obj/item/clothing/shoes/steeltoe (src)
+	new /obj/item/clothing/shoes/steeltoe (src)
+	new /obj/item/clothing/shoes/steeltoe (src)

```

Comment:
If possible could you undo these changes to the gimmick/russian locker, put a standard grey locker there and place all the Russian stuff you want to put on the same tile as that locker. when the world is created the objects will start inside the new locker. for 2 reasons:

1) the gimmick lockers shouldn't be used in the map.
2) the gimmick lockers share the same icon_state as the time machine and that may confuse some folks in game if someone caries the green locker around.